### PR TITLE
Add Visual C++ support

### PR DESCRIPTION
**Pull request by Kazuki-TAKEDA**
I added support for MinGW in the previous Pull Request (#26), but it did not support Visual C++ and lacked tox settings for Windows.
This pull request makes it possible to compile libplinkio with Visual C++ and use tox on Windows.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38-{flake8,black,test,pylint}
+envlist = py38, flake8, black, test, pylint
```
Comment (mfranberg) [inline tox.ini:2]: What is the motivation for this change?

File: src/bim_parse.c [hunk 2]
```diff
@@ -15,6 +15,8 @@
 #include <plinkio/bim.h>
 #include <plinkio/bim_parse.h>
 
+#include <./plinkio_private.h>
```
Comment (mfranberg) [inline src/bim_parse.c:18]:
```suggestion
#include "plinkio_private.h"
```

File: tox.ini (hunk 1 above)
Comment (Kazuki-TAKEDA) [reply tox.ini:2]: Sorry, It is a remnant of testing with Python 3.10.
I will revert it.

File: setup.py [hunk 3]
```diff
@@ -23,80 +26,97 @@
...
-    "plinkio.cplinkio",
-    libplinkio_src_files + libcsv_src_files + pyplinkio_src_files,
-    library_dirs=[],
-    include_dirs=[libplinkio_include_dir, libcsv_include_dir, pyplinkio_include_dir],
-    libraries=[],
-    language="c",
-    extra_compile_args=[],
-    define_macros=[],
-)
+with tempfile.TemporaryDirectory() as tmp_include_dir:
```
Comment (mfranberg) [inline setup.py:29]: This part is a bit errorprone if the snp_lookup.h.in is changed. Perhaps we can pass PLINKIO_BYTE_ORDER as a compiler option instead. Like `-DPLINKIO_BYTE_ORDER=<ENDIAN>` that way we could just pass that to define_macros or extra_compile_args in the python Extension. We could have a 
```c
#ifndef PLINKIO_BYTE_ORDER
#cmakedefine PLINKIO_BYTE_ORDER @PLINKIO_BYTE_ORDER@
#endif
```

to allow the c-library to work correctly. Not sure what the best approach is.

**Comment by mfranberg** [comment]
Thank you for the awesome PR, highly appreciated! I added some comments.

File: setup.py (hunk 3 above)
Comment (Kazuki-TAKEDA) [reply setup.py:33]: I added "#ifndef PLINKIO_BYTE_ORDER" and made the file handling in setup.py robust.

File: src/bim_parse.c (hunk 2 above)
Comment (Kazuki-TAKEDA) [reply src/bim_parse.c]: I fixed it as suggested.

**Comment by Kazuki-TAKEDA** [comment]
I corrected the part where there was your comment.
Would you review my code again, please?

File: tox.ini (hunk 1 above)
Comment (Kazuki-TAKEDA) [reply tox.ini]: I reverted it. (https://github.com/mfranberg/libplinkio/pull/34/commits/72b5ee44d0ec713668a38483252378c6408d8ce5)

File: src/bim_parse.c (hunk 2 above)
Comment (Kazuki-TAKEDA) [reply src/bim_parse.c]: Sorry, it wasn't fixed properly. I fixed it again. (https://github.com/mfranberg/libplinkio/pull/34/commits/7ae141ce816b549462cccee4281b672bceb29dc2)

File: setup.py (hunk 3 above)
Comment (Kazuki-TAKEDA) [reply setup.py:40]: I refactored the file handling code. (https://github.com/mfranberg/libplinkio/pull/34/commits/06a58be1f4f5353ffd0c5e0f4d4b9770c4cdf903)

**Review by mfranberg — APPROVED** [review]

**Comment by mfranberg** [comment]
Looks good to me now, I don't have any environment to test the windows-specific parts so I will trust you on those!
